keyboards/meishi/meishi.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2017 Biacco42
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef MEISHI_H
+#define MEISHI_H
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// The following is an example using the Planck MIT layout
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define KEYMAP( \
```

Comment:
We are trying to not name the layout macros as keymap because it doesn't quite make sense. This is the definition of the structure that a keymap goes into, not the keymap itself.  Can you change `KEYMAP` to `LAYOUT` in this file and in everywhere that uses it in keyboards/meishi. 

```suggestion
#define LAYOUT( \
```